packages/web/components/DataGrid/__tests__/Controls.js

```diff
@@ -0,0 +1,306 @@
+			<Controls
+				data={mockData}
+				handleSortBy={mockHandleSortBy}
+				sortOptions={mockSortOptions}
+				currentOrder={bookmarksEnum.DESC_ORDER}
+			/>,
+		);
+
+		fireEvent.change(getByTestId('select'), {
+			target: { value: 'title' },
+		});
+
+		expect(getByLabelText('Descending Order').firstChild).toHaveClass('active');
+	});
+
+	it('should exec handleSortBy passing option as first argument and undefined as second argument when changing select ', () => {
+		const { getByTestId } = render(
+			<Controls
+				data={mockData}
+				handleSortBy={mockHandleSortBy}
+				sortOptions={mockSortOptions}
+			/>,
+		);
+
+		fireEvent.change(getByTestId('select'), {
+			target: { value: 'title' },
+		});
+
+		expect(mockHandleSortBy.mock.calls).toHaveLength(1);
+		// The first argument of the first call to the function should be 'title'
+		expect(mockHandleSortBy.mock.calls[0][0]).toBe('title');
+		// The second argument of the first call to the function should be 'undefined'
+		expect(mockHandleSortBy.mock.calls[0][1]).toBeUndefined();
+	});
+
+	it('should exec handleSortBy passing current option as first argument and `ASC` as second argument when clicking ascending button ', () => {
+		const { getByLabelText, getByTestId } = render(
+			<Controls
+				data={mockData}
+				handleSortBy={mockHandleSortBy}
+				sortOptions={mockSortOptions}
+			/>,
+		);
+
+		fireEvent.change(getByTestId('select'), {
+			target: { value: 'status' },
+		});
+
+		fireEvent.click(getByLabelText('Ascending Order'));
+
+		expect(mockHandleSortBy.mock.calls).toHaveLength(2);
+		// The first argument of the second call to the function should be 'status'
+		expect(mockHandleSortBy.mock.calls[1][0]).toBe('status');
+		// The second argument of the second call to the function should be 'ASC'
+		expect(mockHandleSortBy.mock.calls[1][1]).toBe(bookmarksEnum.ASC_ORDER);
+	});
+
+	it('should exec handleSortBy passing current option as first argument and `DESC` as second argument when clicking descending button ', () => {
+		const { getByLabelText, getByTestId } = render(
+			<Controls
+				data={mockData}
+				handleSortBy={mockHandleSortBy}
+				sortOptions={mockSortOptions}
+			/>,
+		);
+
+		fireEvent.change(getByTestId('select'), {
+			target: { value: 'id' },
+		});
+
+		fireEvent.click(getByLabelText('Descending Order'));
+
+		expect(mockHandleSortBy.mock.calls).toHaveLength(2);
+		// The first argument of the second call to the function should be 'id'
+		expect(mockHandleSortBy.mock.calls[1][0]).toBe('id');
+		// The second argument of the second call to the function should be 'DESC'
+		expect(mockHandleSortBy.mock.calls[1][1]).toBe(bookmarksEnum.DESC_ORDER);
+	});
+
+	it('should render pagination controls if handlePagination is not null', () => {
+		const { getByLabelText } = render(
+			<Controls data={mockData} handlePagination={mockHandlePagination} />,
+		);
+
+		expect(getByLabelText('First Page')).toBeInTheDocument();
```

Comment:
Se você usar o `screen` do react testing library você não precisa pegar o `getByLabelText`.

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen